readme.md

```diff
@@ -84,8 +97,15 @@ $service->loadIdentifier('Authentication.Orm', [
 ]);
 
 // Load the authenticators leaving Basic as the last one.
-$service->loadAuthenticator('Authentication.Session');
-$service->loadAuthenticator('Authentication.Form');
+$service->loadAuthenticator('Authentication.Session', [
+    'sessionKey' => 'Auth'
```

Comment:
Yep, I thought it was worth showing the configuration key in the documentation example, so that people know they can customise it. Perhaps it's worth following the other examples, and using a custom value with a comment for the default?